pkg/apis/certmanager/v1alpha2/types_certificaterequest.go

```diff
@@ -23,9 +23,16 @@ import (
 )
 
 const (
+	// Pending indicates that a CertificateRequest is still in progress.
 	CertificateRequestReasonPending = "Pending"
-	CertificateRequestReasonFailed  = "Failed"
-	CertificateRequestReasonIssued  = "Issued"
+
+	// Failed indicates that a CertificateRequest has failed, either due to
+	// timing out or some other critical failure.
```

Comment:
```suggestion
	// CertificateRequestReasonFailed indicates that a CertificateRequest has failed, either due to
	// timing out or some other critical failure.
```

Reply:
I don't think this is a useful change, so "resolving" this 😄